packages/near-api-js/test/account.access_key.test.js

```diff
@@ -40,8 +40,8 @@ test('remove access key no longer works', async() => {
         await contract.setValue({ args: { value: 'test' } });
         fail('should throw an error');
     } catch (e) {
-        expect(e.message).toEqual(`Can not sign transactions for account ${workingAccount.accountId} on network ${testUtils.networkId}, no matching key pair exists for this account`);
-        expect(e.type).toEqual('KeyNotFound');
+        expect(e.message).toContain(`Querying [object Object] failed: access key ${publicKey} does not exist while viewing.`);
```

Comment:
`[object Object]` isn't that helpful, should we stringify it instead? That might bloat the error message though, maybe there's a better way we can structure the message overall?

Reply:
Maybe we may leave it as `Query failed: access key ...` since later it prints `${publicKey} does not exist while viewing.`

Reply:
Sounds good 👍🏽